text_generation/causal_lm/cpp/README.md

```diff
@@ -150,5 +150,5 @@ To enable Unicode characters for Windows cmd open `Region` settings from `Contro
 10. [zephyr-7b-beta](https://huggingface.co/HuggingFaceH4/zephyr-7b-beta)
 11. [redpajama-3b-chat](https://huggingface.co/ikala/redpajama-3b-chat)
 12. [Mistral-7B-v0.1](https://huggingface.co/mistralai/Mistral-7B-v0.1)
-
+13. [Gemma-2B-it](https://huggingface.co/google/gemma-2b-it)
```

Comment:
Is it possible to add the model to the tests?

Reply:
gemma has gated repo with requirements to share personal info for getting access. Can testing model with random weights used instead? If yes, I suggest to use https://huggingface.co/Xenova/tiny-random-GemmaForCausalLM

Reply:
The model with random weights doesn't have a license. Most likely I'll merge this PR as is without tests. I'll get back tomorrow with a definite desision.

Reply:
I am syncing this branch with the master.. If that's not a problem... because it is outdated